### PR TITLE
[MGP-473] add flights live pricing v3 migration guide

### PR DESCRIPTION
**Pull request by louiskueh**
Adds the migration guide required for flights live pricing v3. Decided to do it in slate since we have it already and v2 users will see it there. It will be easy to migrate to the future docs since it's in markdown.

Do comment if you think something is missing or have a good idea to add!

* Overview
* Migration checklist
* Endpoint mapping
* Request mapping
* Response mapping

Migration guide screenshot
image

Response mapping screenshot
image

Appreciate it is hard to read, so have uploaded the md files here
* [Migration guide](https://github.skyscannertools.net/louis-kueh/migration-guide/blob/main/migration-guide.md)
* [Response mapping](https://github.skyscannertools.net/louis-kueh/migration-guide/blob/main/response-mapping.md)

Note that code blocks will appear on the right in slate

### CODE REVIEW

File: source/includes/Migration_guide_v3_flights_live_pricing.md.erb [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+And upcoming features:
+
+* Fare families
+* Multi-city
+* Booking calls support (itinerary pricing options)
+* total cost (baggages, cancellation, advance charge ) info
+* Flights Explore (previously browse)
+* hotels
+* car hire
```
Comment (owenmerry) [inline source/includes/Migration_guide_v3_flights_live_pricing.md.erb:21]: Not sure about explaining features that are not built yet, specially so high in the doc as it just confuses the reader

File: source/includes/Migration_guide_v3_flights_live_pricing.md.erb [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+
+b2b-partners-api V3 will provide feature-parity as https://www.skyscanner.net/. This currently includes:
+
+* dBook / sBook content
+* Greener choices
+* PQS data
+* FQS data
+* Mash-ups
+* Fare basis code
+* Regulatory info for FR
```
Comment (owenmerry) [inline source/includes/Migration_guide_v3_flights_live_pricing.md.erb:11]: I feel a table and description of what the feature is is also nessary as an external user would'nt. know alot of these

File: source/includes/Migration_guide_v3_flights_live_pricing.md.erb [hunk 3]
```diff
@@ -0,0 +1,163 @@
+# V3 migration guide (Flights Live Pricing)
```
Comment (owenmerry) [inline source/includes/Migration_guide_v3_flights_live_pricing.md.erb:1]: i think we might need to think of a better way to name this as the side menu cuts off alot of the description

Screenshot 2022-06-14 at 13 49 52

maybe
```
# Migration Guide
```

also we want to do upper case for the title like other menu items

File: source/includes/Migration_guide_v3_flights_live_pricing_response_mappings.md.erb [hunk 4]
```diff
@@ -0,0 +1,568 @@
...
+            ]
+          }
+        ]
+      }
+    ],
+    "legIds": [
+      "11235-2301201130--32480-0-13554-2301201300",
+      "13554-2301211130--32480-0-11235-2301211255"
+    ],
+    "greenerChoicesData": null
```
[the remark this answers is not in the document]
Comment (tudorscanner) [reply source/includes/Migration_guide_v3_flights_live_pricing_response_mappings.md.erb:139]: I renamed this in this PR (not merged yet)

https://github.skyscannertools.net/b2b-platform/b2b-partners-api/pull/112

**Pull request closed without merging**